integration_test/cleanup/cleanup-orca.sh

```diff
@@ -6,7 +6,12 @@ set -ex
 source integration_test/shared/orca-terraform.sh
 
 # Destroy orca via terraform
+cd cumulus-orca
+# todo: Only build once. Reuse for various stages/jobs
```

Comment:
You have a lot of little todo items throughout this and other files in this PR. You will want to make sure there are Jira Cards that correspond to them and possibly notate the Jira card number in the code here so that work can be prioritized.

Reply:
Done.